web/src/data/events.js

```diff
@@ -31,6 +41,11 @@ const defaultDyosSchedule = [
   { time: "9:00pm - 11:30pm", name: "Social Dancing" },
 ];
 
+const addons = [
```

Comment:
What is this? Looks like it's not used anywhere

Reply:
Oh I thought I removed this